fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/POSITION/MfdFmsPositionNavaids.tsx

```diff
@@ -590,7 +565,7 @@ export class MfdFmsPositionNavaids extends FmsPage<MfdFmsPositionNavaidsProps> {
                 <InputField<number>
                   dataEntryFormat={new LsCourseFormat()}
                   dataHandlerDuringValidation={async (v) => {
-                    this.props.fmcService.master?.navaidTuner.setIlsCourse(v || null);
+                    this.props.fmcService.master?.navaidTuner.setIlsCourse(v || null, v && v < 0 ? true : false);
```

Comment:
setIlsCourse expects [0, 360].
```suggestion
                    this.props.fmcService.master?.navaidTuner.setIlsCourse(v !== null ? Math.abs(v) : null, v && v < 0 ? true : false);
```

Reply:
Yeah, it felt dirty but it seems it's the only way we can do it atm.